README.md

```diff
@@ -1,5 +1,7 @@
 # Hermione
 
+> ⚠️ This project has been renamed to [testplane](https://www.npmjs.com/package/testplane), which is backward compatible and you can migrate to it very quickly. More info [here](https://github.com/gemini-testing/testplane/tree/master?tab=readme-ov-file#rename-from-hermione).
```

Comment:
quickly → swiftly

Reply:
swiftly -> rapidly

All these words are synonymous or you think it sounds better with `swiftly` word? )